inst/rmarkdown/templates/lipics_article/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,309 @@
+
+- Do not use a different main font. (For example, the `times` package is forbidden.)
+- Do not alter the spacing of the `lipics-v2019.cls` style file.
+- Do not use `enumitem` and `paralist`. (The `enumerate` package is preloaded, so you can use
+ `\begin{enumerate}[(a)]` or the like.)
+- Do not use "self-made" sectioning commands (e.\,g., `\bf My Paragraph}`).
+- Do not hide large text blocks using comments or `\iffalse` $\ldots$ `\fi` constructions.
+- Do not use conditional structures to include/exclude content. Instead, please provide only the content that should be published -- in one file -- and nothing else.
+- Do not wrap figures and tables with text. In particular, the package `wrapfig` is not supported.
+- Do not change the bibliography style. In particular, do not use author-year citations. (The
+`natbib` package is not supported.)
+
+\enlargethispage{\baselineskip}
+
+This is only a summary containing the most relevant details. Please read the complete document "LIPIcs: Instructions for Authors and the `lipics-v2019` Class" for all details and don't hesitate to contact Dagstuhl Publishing ([mailto:publishing@dagstuhl.de](publishing@dagstuhl.de)) in case of questions or comments:
+[http://drops.dagstuhl.de/styles/lipics-v2019/lipics-v2019-authors/lipics-v2019-authors-guidelines.pdf](`http://drops.dagstuhl.de/styles/lipics-v2019/lipics-v2019-authors/lipics-v2019-authors-guidelines.pdf).
+
+\section{Lorem ipsum dolor sit amet}
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit \cite{DBLP:journals/cacm/Knuth74}. Praesent convallis orci arcu, eu mollis dolor. Aliquam eleifend suscipit lacinia. Maecenas quam mi, porta ut lacinia sed, convallis ac dui. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse potenti. Donec eget odio et magna ullamcorper vehicula ut vitae libero. Maecenas lectus nulla, auctor nec varius ac, ultricies et turpis. Pellentesque id ante erat. In hac habitasse platea dictumst. Curabitur a scelerisque odio. Pellentesque elit risus, posuere quis elementum at, pellentesque ut diam. Quisque aliquam libero id mi imperdiet quis convallis turpis eleifend.
+
+\begin{lemma}[Lorem ipsum]
+\label{lemma:lorem}
+Vestibulum sodales dolor et dui cursus iaculis. Nullam ullamcorper purus vel turpis lobortis eu tempus lorem semper. Proin facilisis gravida rutrum. Etiam sed sollicitudin lorem. Proin pellentesque risus at elit hendrerit pharetra. Integer at turpis varius libero rhoncus fermentum vitae vitae metus.
+\end{lemma}
+
+\begin{proof}
+Cras purus lorem, pulvinar et fermentum sagittis, suscipit quis magna.
+
+\begin{claim}
+content...
+\end{claim}
+\begin{claimproof}
+content...
+\end{claimproof}
+
+\end{proof}
+
+\begin{corollary}[Curabitur pulvinar, \cite{DBLP:books/mk/GrayR93}]
```

Comment:
@cderv Should this `corollary` also be possible with the new filter?

Reply:
Yes this would do it
```md
:::{.corollary data-latex="[Curabitur pulvinar, \cite{DBLP:books/mk/GrayR93}]"}
\label{lemma:curabitur}
Nam liber tempor cum soluta nobis eleifend option congue nihil imperdiet doming id quod mazim placerat facer possim assum. Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat.
:::
```
➡️ 
```tex
\begin{corollary}[Curabitur pulvinar, \cite{DBLP:books/mk/GrayR93}]

\label{lemma:curabitur} Nam liber tempor cum soluta nobis eleifend
option congue nihil imperdiet doming id quod mazim placerat facer possim
assum. Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed
diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat
volutpat.

\end{corollary}
```

Reply:
Done!